src/Core/Repositories/SqlServer/SsoConfigRepository.cs

```diff
@@ -57,5 +57,17 @@ public async Task<ICollection<SsoConfig>> GetManyByRevisionNotBeforeDate(DateTim
                 return results.ToList();
             }
         }
+
+        public override async Task CreateAsync(SsoConfig obj)
+        {
+            obj.CreationDate = obj.RevisionDate = DateTime.UtcNow;
+            await base.CreateAsync(obj);
+        }
+
+        public override async Task ReplaceAsync(SsoConfig obj)
+        {
+            obj.RevisionDate = DateTime.UtcNow;
+            await base.ReplaceAsync(obj);
+        }
```

Comment:
A proper fix here would be to introduce an SSoConfig service. The repositories shouldn't really be doing this IMO. You could create a save method that set's the creation/revision dates